docs/boostrap.md

```diff
@@ -14,28 +14,51 @@
 <a href="https://github.com/mainflux/sdk-py/blob/main/mainflux/boostrap.py#L8"><img align="right" style="float:right;" src="https://img.shields.io/badge/-source-cccccc?style=flat-square"></a>
 
 ## <kbd>class</kbd> `Bootstrap`
+Bootstrap service API client. 
 
+Bootstrap service is used to manage configurations for Mainflux Things. It provides  services such as updating, viewing, removing and adding new configurations. 
 
 
 
-<a href="https://github.com/mainflux/sdk-py/blob/main/mainflux/boostrap.py#L14"><img align="right" style="float:right;" src="https://img.shields.io/badge/-source-cccccc?style=flat-square"></a>
+**Attributes:**
+ 
+ - <b>`url`</b> (str):  Mainflux Bootstrap API URL. 
```

Comment:
Why do we use HTML tags `<b>` in the Markdown? Can't we do it with `*`?

Reply:
These come with lazydocs generation. But I've tried with * and all it does is italicize the word. 
<img width="288" alt="image" src="https://github.com/mainflux/sdk-py/assets/115026536/25ad8d19-3cee-4efc-8e07-1693c0ae96ad">

Reply:
Please do not resolve by hand. Reviewer will resolve once the remark is really resolved.

This one is not resolves - please proceed and fix Markdown as explained.